packages/notebook-extension/src/index.tsx

```diff
@@ -1060,6 +1065,87 @@ const activeCellTool: JupyterFrontEndPlugin<void> = {
   }
 };
 
+export const filterCellsPlugin: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/notebook-extension:filter-cells',
+  description: 'Filter cells.',
```

Comment:
Maybe rename the plugin to `:cell-filter` instead?

Reply:
I have chosen `:cell-filters` and `cellFilters` everywhere it is relevant.